config/lpdc-management/characteristics/form.ttl

```diff
@@ -198,6 +198,13 @@ ext:authorityPg a form:PropertyGroup;
       form:help "Meerdere overheden kunnen worden toegewezen";
       sh:order 30 ;
       sh:path lpdcExt:hasExecutingAuthority ;
+      form:validations [
+        a form:RequiredConstraint ;
+        form:language "en";
```

Comment:
This is only needed when validating something that expects a lang string. 
In our case this is a code list entry.

Reply:
So remove this